test/Interop/Cxx/extern-var/extern-var-irgen.swift

```diff
@@ -0,0 +1,36 @@
+// RUN: %empty-directory(%t)
+// RUN: %target-build-swift %s -I %S/Inputs -emit-ir -o %t/extern-var.ir -Xfrontend -enable-cxx-interop
+// RUN: %FileCheck < %t/extern-var.ir %s
+
+// CHECK: @counter = external global i32, align 4
+
+// CHECK: define hidden swiftcc i32 @"$s4main10getCounters5Int32VyF"() #0
```

Comment:
Usually we interleave the input code and the CHECK lines for readability -- I suggest doing the same here.

Reply:
Done.